OracleWebLogic/dockerfiles/14.1.1.0/Dockerfile.developer

```diff
@@ -0,0 +1,108 @@
+# Copyright 2020, Oracle Corporation and/or its affiliates. 
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for WebLogic 14.1.1.0 Quick Install Distro
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) fmw_14.1.1.0.0_wls_quick_Disk1_1of1.zip
+#     Download the Developer Quick installer from http://www.oracle.com/technetwork/middleware/weblogic/downloads/wls-for-dev-1703574.html
+#
+# (2) jdk-11.0.6_linux-x64_bin.tar.gz 
+#     Download from http://www.oracle.com/technetwork/java/javase/downloads/jdk-11.0.6_linux-x64_bin.tar.gz 
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run:
+#      $ docker build -f Dockerfile.developer -t oracle/weblogic:14.1.1.0-developer .
+#
+# IMPORTANT
+# ---------
+# The resulting image of this Dockerfile contains a WLS Empty Domain.
+#
+# From the OIracle Docker GitHub Project
```

Comment:
Typo. Also, it's probably better referred to as the "Oracle `docker-images` repository on GitHub."